eng/pipelines/templates/steps/analyze.yml

```diff
@@ -26,6 +26,17 @@ steps:
       python find_invalid_versions.py --always-succeed --service=${{parameters.ServiceDirectory}}
     displayName: Find Invalid Versions
 
+  - pwsh: |
+      Get-ChildItem .\ -Filter "*.py" |
+      Foreach-Object {
+        (Get-Content $_ -Raw) -match "\r\n$" && Write-Output $_
+      }
+      Get-ChildItem .\ -Filter "*.md" |
+      Foreach-Object {
+        (Get-Content $_ -Raw) -match "\r\n$" && Write-Output $_
+      }
+    displayName: Check for CRLF Line endings
```

Comment:
Did you want this step to fail if you find any files with CRLF?

Reply:
Yes @weshaggard

Reply:
I added a `Throw` chaining to these two statements

Reply:
You can just call exit 1 as well and that will cause the step to fail.